portal/src/main/java/com/liferay/faces/portal/render/internal/PortalTagRendererCompat.java

```diff
@@ -28,8 +28,12 @@
  */
 public abstract class PortalTagRendererCompat extends Renderer {
 
+	// Public Constants
+	public static final String AUI_FORM_USE_NAMESPACE = "aui:form:useNamespace";
+	public static final String AUI_FORM_PORTLET_NAMESPACE = "aui:form:portletNamespace";
```

Comment:
@juangon OK, since they are only used in one place, I just inlined them in a code review commit: https://github.com/liferay/liferay-faces-portal/commit/73a9423d854e4a4dec510db32ffc3254fc3a9ecd